Review lesson with v3.6

katrinleinweber opened the pull request:
@RealLinna & me will soon host [a workshop](https://konferenz.uni-hannover.de/event/49/contributions/376/) with this lesson. While preparing, I noticed these slightly outdated items and suggest the relevant changes here.

No show-stoppers so far, in line with the only _minor_ OpenRefine update 🤞

- [x] review `assets/img/openrefine_ui.png` whether an update is needed

katrinleinweber commented:
Hi @rhizomed & @ostephens 👋 Could you please review these changes? They resolve [#210 (New OR version, impacts on lessons?)](https://github.com/LibraryCarpentry/lc-open-refine/issues/210).

ostephens commented on _extras/guide.md:
```diff
@@ -36,7 +36,7 @@ ____
 
 * If a learner is unable to install OpenRefine on their computer due to IT restrictions for example, there are cloud services available that they could try:
-  - [openrefineder](https://github.com/betatim/openrefineder/) using MyBinder [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/betatim/openrefineder/6ba108b?urlpath=%2Fopenrefine) (OpenRefine 3.4.1, free to use without registration, [restricted](https://mybinder.readthedocs.io/en/latest/faq.html#how-much-memory-am-i-given-when-using-binder) to 1-2 GB RAM and server will be d ...
+  - [openrefineder](https://github.com/betatim/openrefineder/) using MyBinder [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/betatim/openrefineder/6ba108b?urlpath=%2Fopenrefine) (free to use without registration, [restricted](https://mybinder.readthedocs.io/en/latest/faq.html#how-much-memory-am-i-given-when-using-binder) to 1-2 GB RAM and server will be deleted after 10 mi ...
```
I think it's important to note that if you use the MyBinder version it's OpenRefine 3.4.1 rather than the current one so I think I'd prefer to leave this specific version mentioned

jas58 replied:
This might also be a useful instructor note.  Some workshops need to use the MyBinder version rather than local installs.

katrinleinweber replied:
OK, amended offline.

ostephens commented on index.md:
```diff
@@ -4,15 +4,6 @@ layout: lesson
-> 
-> OpenRefine has some issues with Firefox. Internet Explorer is not supported. 
-> 
+> To complete this lesson you will need to install OpenRefine and download a data file.
```
I think it makes sense to keep the download links here in the text. Happy to discuss further the pros and cons but it's not a factor in the OpenRefine version so think we could deal with that discussion separately

jas58 replied:
We might need to check with Brynne, the new Accessibility Coordinator.  If the link appears several times on the same page, screenreaders become painful.  Maybe refer to the Setup page?

katrinleinweber replied:
Let's move this part into https://github.com/LibraryCarpentry/lc-open-refine/pull/248

ostephens commented on _episodes/11-using-arrays-transformations.md:
````diff
@@ -60,7 +60,7 @@ Taking the above example again, this would result in a string with the days of t
 >2. On the Authors column, use the dropdown menu and select ```Facet->Custom text facet...```
 >3. The ```Custom text``` facet function allows you to write GREL functions to create a facet
->4. In the Expression box type ```value.contains(",").toString()```
+>4. In the Expression box type ```value.contains(",")```
````
Removing the 'toString()' here will prevent the lesson working with OpenRefine 3.1. Leaving it in makes the instruction work in all versions of OpenRefine. I don't have an issue with removing it but just noting in case you think it's worth leaving in

katrinleinweber replied:
Hm, true… However, that version is [almost 4 years old](https://github.com/OpenRefine/OpenRefine/releases/tag/3.1), and at least 2 security vulnerabilities were fixed since then.

I'd argue that one more nudge to get people to update is OK.

ostephens commented on _episodes/10-data-transformation.md:
````diff
@@ -59,7 +59,7 @@ Such tests can be combined with other GREL expressions to create more complex tr
 >2. On the Authors column, use the dropdown menu and select ```Facet->Custom text facet...```
 >3. The Custom text facet function allows you to write GREL functions to create a facet
->4. In the Expression box type ```value.contains(",").toString()```
+>4. In the Expression box type ```value.contains(",")```
````
Removing the 'toString()' here will prevent the lesson working with OpenRefine 3.1. Leaving it in makes the instruction work in all versions of OpenRefine. I don't have an issue with removing it but just noting in case you think it's worth leaving in

katrinleinweber replied:
[See above](https://github.com/LibraryCarpentry/lc-open-refine/pull/242#discussion_r962266503)

ostephens commented on setup.md:
```diff
@@ -39,10 +39,10 @@ to download an installer package. Please note that
 `sudo apt install default-jre`.
-* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/), 
+* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/),
+[Microsoft Edge](https://www.microsoft.com/edge),
```
The statement from the OpenRefine documentation says:

> OpenRefine works best on browsers based on Webkit, such as:
> * Google Chrome
> * Chromium
> * Opera
> * Microsoft Edge
> We are aware of some minor rendering and performance issues on other browsers such as Firefox. We don't support Internet Explorer.

I wonder if we might just re-use this wording as it seems clear and detailed

jas58 replied:
I agree this Webkit language is very useful.  Should we use it in the index.md section  (as a later issue because you mentioned it's less crucial to this PR).

katrinleinweber replied:
Let's move this part into https://github.com/LibraryCarpentry/lc-open-refine/pull/248

ostephens reviewed (changes requested):
Additionally the new openrefine_ui.png doesn't have highlighting on the "Create project" button

ostephens commented:
Thanks so much @katrinleinweber for this thorough review and all your work - it is really appreciated. Hope my comments and change requests here are helpful

katrinleinweber commented:
Thanks for your review @ostephens & @jas58. I've moved the unrelated setup changes into https://github.com/LibraryCarpentry/lc-open-refine/pull/248 and addressed your comments with an updated set of commits.

ostephens commented:
Thanks so much @katrinleinweber - hugely helpful. Hope the upcoming workshop goes well
